SmartDeviceLink/SDLIAPSession.m

```diff
@@ -24,13 +31,23 @@ @implementation SDLIAPSession
 - (instancetype)initWithAccessory:(EAAccessory *)accessory forProtocol:(NSString *)protocol {
     NSString *logMessage = [NSString stringWithFormat:@"SDLIAPSession initWithAccessory:%@ forProtocol:%@", accessory, protocol];
     [SDLDebugTool logInfo:logMessage];
+    
+    if ([protocol isEqualToString:@"com.smartdevicelink.prot0"]){
+        _isDataSession = NO;
+    }
+    else{
```

Comment:
Please note the objective-c style guide [for this project](https://github.com/smartdevicelink/sdl_ios/wiki/Objective-C-Style-Guide). The else should be on the same line as the `}`.